public/js/calendar.js

```diff
@@ -2,7 +2,7 @@ var Cal = function (divId) {
   this.divId = divId;
 
   // Days of week, starting on Sunday
-  this.DaysOfWeek = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+  this.DaysOfWeek = ['S', 'M', 'T', 'W', 'T', 'F', 'S'];
```

Comment:
I would say having the days 3 letters is clearer bit this looks better i guess.

Reply:
Yea this was in the design that some legend made so

Reply:
But yea I don't think anyone would have any trouble working it out

Reply:
> Yea this was in the design that some legend made so
 LOL